### PR TITLE
#67 Create Custom Cursor

### PR DESCRIPTION
**Pull request by Mikivitkovski**

### CODE REVIEW

**Comment by edichoska** [comment]
@kire21  can you please help @Mikivitkovski with resolving these conflicts, so nothing is lost. thanks 🙏

**Comment by kire21** [comment]
> @kire21  can you please help @Mikivitkovski with resolving these conflicts, so nothing is lost. thanks 🙏 

OK. I'll be in touch with @Mikivitkovski

File: components/custom-cursor/customcursor.module.scss [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  border-radius: 50%;
+  z-index: 9999;
+  pointer-events: none;
+  mix-blend-mode: difference;
+  transition:
+    width 0.5s ease,
+    height 0.5s ease;
+}
+
+.large {
```
Comment (stoimenov95) [inline components/custom-cursor/customcursor.module.scss:15]: @Mikivitkovski we can name this `.largeCursor`

**Review by stoimenov95 — CHANGES_REQUESTED** [review]
I have nothing else to add, except for the class name change.

Kudos to you, @Mikivitkovski, the code is very well structured, optimized, and performant.

However, @edichoska, should we be worried about the site's accessibility?

**Comment by stoimenov95** [comment]
@Mikivitkovski did you come up with a compromise with @edichoska to use the `square` icon for the cursor in the blog section?

**Comment by Mikivitkovski** [comment]
> @Mikivitkovski did you come up with a compromise with @edichoska to use the `square` icon for the cursor in the blog section?

Yes we speak, and she told me to be just arrow, and i send to her more different arrows in cursor to decide how its better look, and she told me to do it like that one in the commit

**Review by stoimenov95 — APPROVED** [review]

**Pull request merged**